src/halmos/__main__.py

```diff
@@ -583,10 +583,8 @@ def select(var):
         elif args.verbose >= 1:
             if name.startswith('storage') or name.startswith('msg_') or name.startswith('this_'): return True
         return False
-    if args.debug:
-        return str(model)
-    else:
-        return '[' + ', '.join(sorted(map(lambda decl: f'{decl} = {model[decl]}', filter(select, model)))) + ']'
+    select_model = filter(select, model) if not args.debug else model
+    return '[' + f',\n{" "*17}'.join(sorted(map(lambda decl: f'{decl} = {hexify(model[decl])}', select_model))) + ']'
```

Comment:
17 spaces? that's oddly specific

Reply:
ah, yeah, i was trying to print counterexamples like:
```
Counterexample: [x = 0x1,
                 y = 0x2,
                 z = 0x3]
Counterexample: [x = 0x11,
                 y = 0x22,
                 z = 0x33]
```

multi-line is used to better handle the cases with many variables (which happen in real examples).

please feel free to suggest if you have a better idea to print these!

Reply:
now it prints:
```
Counterexample: [
    x = 0x1,
    y = 0x2,
    z = 0x3]
```